docker-compose.yml

```diff
@@ -38,16 +38,16 @@ geodata:
   ports:
     - "2222:22"
   volumes:
-    - geoserver_geodata:/home/geoserver/data
+    - geoserver_geodata:/mnt/geoserver_geodata
 
 # geOrchestra images
     
 geoserver:
   image: georchestra/geoserver:latest
   volumes:
-    - geoserver_datadir:/var/local/geoserver
-    - geoserver_geodata:/var/local/geodata
-    - geoserver_tiles:/var/local/tiles
+    - geoserver_datadir:/mnt/geoserver_datadir
+    - geoserver_geodata:/mnt/geoserver_geodata
+    - geoserver_tiles:/mnt/geoserver_tiles
```

Comment:
This should be updated https://github.com/georchestra/georchestra/blob/master/geoserver/webapp/src/docker/Dockerfile#L21 ?